README.md

```diff
@@ -54,17 +54,17 @@ helm install "${SKYWALKING_RELEASE_NAME}" \
   oci://registry-1.docker.io/apache/skywalking-helm \
   --version "${SKYWALKING_RELEASE_VERSION}" \
   -n "${SKYWALKING_RELEASE_NAMESPACE}" \
-  --set oap.image.tag=10.1.0 \
+  --set oap.image.tag=10.3.0 \
   --set oap.storageType=banyandb \
-  --set ui.image.tag=10.1.0 \
+  --set ui.image.tag=10.3.0 \
   --set elasticsearch.enabled=false \
   --set banyandb.enabled=true \
-  --set banyandb.image.tag=0.7.0
+  --set banyandb.image.tag=0.9.0
 ```
 
 BanyanDB can be configured through various parameters. A comprehensive list of these parameters can be found in the configuration section of [BanyanDB Helm](https://github.com/apache/skywalking-banyandb-helm?tab=readme-ov-file#configuration) repository. These parameters allow you to customize aspects such as replication, resource allocation, persistence, and more to suit your specific deployment needs. Remember to prepend 'banyandb.' to all parameter names when applying the settings. For example, `banyandb.image.tag` can be used to specify the version of BanyanDB.
 
-The _BanyanDB(>=0.7.0)_ is not compatible with the _BanyanDB Helm Chart(<0.3.0)_ which was referred by the _SkyWalking Helm Chart(<4.7.0)_. Meanwhile, _BanyanDB(>=0.7.0)_ requires OAP 10.1.0+. If you want to use _BanyanDB(>=0.7.0)_ as storage solution, you have to use the _BanyanDB Helm Chart(>=0.3.0)_ or _SkyWalking Helm Chart(>=4.7.0)_.
+The _BanyanDB(>=0.7.0)_ is not compatible with the _BanyanDB Helm Chart(<0.3.0)_ which was referred by the _SkyWalking Helm Chart(<4.7.0)_. Meanwhile, _BanyanDB(>=0.7.0)_ requires OAP 10.3.0+. If you want to use _BanyanDB(>=0.7.0)_ as storage solution, you have to use the _BanyanDB Helm Chart(>=0.3.0)_ or _SkyWalking Helm Chart(>=4.7.0)_.
```

Comment:
```suggestion
```

As we have bundled db-helm with main helm, I think we don't need this, as the user will use them automatically.